### ISSUE TITLE
AppVoyer configuration issue

### ISSUE THREAD

**Issue by lupengamzn** [aws/aws-xray-sdk-dotnet#172]
AppVoyer will build sdk without [signing](https://github.com/aws/aws-xray-sdk-dotnet/blob/master/appveyor.yml#L12) the assemblies, in this case, `dotnet cli` will override project's default building properties and unit test library will not have accessibility to the internal class/APIs without a strong name signature, leading to a building failure. 

CC: @srprash @yogiraj07

**Comment by lupengamzn** [comment]
#171 met this issue
